db/dcrpg/queries.go

```diff
@@ -1455,6 +1524,32 @@ func RetrieveBlockSummaryByTimeRange(db *sql.DB, minTime, maxTime int64, limit i
 	return blocks, nil
 }
 
+// retrieveTicketsTxsBlockTime fetches the block time for all the tickets
+// transactions in the transactions table. The Tx struct from this function
+// only contains the block height and the block time.
+func retrieveTicketsTxsBlockTime(db *sql.DB) ([]*dbtypes.Tx, error) {
+	var txsData = make([]*dbtypes.Tx, 0)
```

Comment:
Just `var txsData []*dbtypes.Tx` please, unless the caller really needs a non-nil zero-length slice for some reason.

Reply:
ok. Noted